pkg/porter/lifecycle_test.go

```diff
@@ -585,7 +586,7 @@ func Test_ensureVPrefix(t *testing.T) {
 				assert.False(t, tt.args.opts._ref.Tag()[0] == 'v')
 			}
 
-			err := ensureVPrefix(tt.args.opts)
+			err := ensureVPrefix(tt.args.opts, os.Stdout)
```

Comment:
Do we need to make all this noise in the test?  Can we use `io.Discard`?

Reply:
good idea!